.ci/continuous/Jenkinsfile

```diff
@@ -42,6 +55,46 @@ pipeline {
             }
         }
 
+        stage('Snyk Security') {
+            when {
+                expression { gitUtils.isReleaseTag() || gitUtils.isReleaseCandidate() || gitUtils.isReleaseBranch() }
```

Comment:
your going to need isMasterBranch() here due to the branching strategy of this repo .

i'd leave isReleaseBranch in case release/* pattern is used here in future